<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QLabel, QLineEdit, QPushButton{
	font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,0">
    <item>
     <widget class="QwtPlot" name="qwtPlot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QwtPlot" name="qwtPlotTwo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>每秒采样次数</string>
        </property>
        <property name="text">
         <string>X轴宽度[s]:</string>
        </property>
        <property name="buddy">
         <cstring>leSamplingRate</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leXAxisWidth">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>绘制间隔时间:</string>
        </property>
        <property name="buddy">
         <cstring>leIntervalTime</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePlotIntervalTime">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>每秒采样次数</string>
        </property>
        <property name="text">
         <string>采样率:</string>
        </property>
        <property name="buddy">
         <cstring>leSamplingRate</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSamplingRate">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>33</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>间隔时间:</string>
        </property>
        <property name="buddy">
         <cstring>leIntervalTime</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leIntervalTime">
        <property name="maximumSize">
         <size>
          <width>119</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfirmModify">
        <property name="text">
         <string>确认修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string>每秒采样次数</string>
         </property>
         <property name="text">
          <string>区域一绘制长度[小时]:</string>
         </property>
         <property name="buddy">
          <cstring>leSamplingRate</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leDrawLength">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDrawHistory">
         <property name="text">
          <string>开始绘制</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="toolTip">
          <string>每秒采样次数</string>
         </property>
         <property name="text">
          <string>区域二绘制长度[小时]:</string>
         </property>
         <property name="buddy">
          <cstring>leSamplingRate</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leSecondDrawLength">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSecondDrawHistory">
         <property name="text">
          <string>开始绘制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddMarker">
         <property name="text">
          <string>添加标记</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
